README.md

```diff
@@ -10,14 +10,207 @@ support new services is also a fairly simple process.
+
+# PRESCIENT
+lochness_create_template.py \
+    --outdir /data/lochness_root \
+    --studies BWH McLean \
+    --sources RPMS daris mediaflux mindlamp \
+    --email kevincho@bwh.harvard.edu \
+    --poll_interval 43200 \
+    --ssh_host erisone.partners.org \
+    --ssh_user kc244 
+
+# For more options: lochness_create_template.py -h
+```
+
+
+### Making edits to the template
+
+Running one of the commands above will create the structure below
+
+```sh
+/data/lochness_root/
+├── 1_encrypt_command.sh
+├── 2_sync_command.sh
+├── PHOENIX
+│   ├── GENERAL
+│   │   ├── BWH
+│   │   │   └── BWH_metadata.csv
+│   │   └── McLean
+│   │       └── McLean_metadata.csv
+│   └── PROTECTED
+│       ├── BWH
+│       └── McLean
+├── config.yml
+├── lochness.json
+└── pii_convert.csv
+```
+
+
+
+1. Change information in `config.yml` and `lochness.json` as needed.
+
+
+2. Either manually update the `PHOENIX/GENERAL/*/*_metadata.csv` or
+   amend the field names in REDCap / RPMS sources correctly for lochness to
+   automatically update the metadata files.
+
+   Currently, lochness initializes the metadata using the following field names 
+   in REDCap and RPMS.
+
+- `record_id1`: the record ID field name
+- `Consent`: the field name of the consent date
+- `beiwe_id`: the field name of the BEIWE ID.
+- `xnat_id`: the field name of the XNAT ID.
+- `dropbox_id`: the field name of the Dropbox ID.
+- `box_id`: the field name of the Box ID.
+- `mediaflux_id`: the field name of the Mediaflux ID.
+- `mindlamp_id`: the field name of the Mindlamp ID.
+- `daris_id`: the field name of the DaRIS ID.
+- `rpms_id`: the field name of the RPMS ID.
+
+
+3. Encrypt the `lochness.json` by running ::
+
+```sh
+bash 1_encrypt_command.sh
+```
+
+This encryption step creates a copy of encrypted keyrings to
+`/data/lochness_root/.lochness.enc`. To protect the sensitive keyring
+information in json, remove the `lochness.json` after running the encryption.
+
+
+You can still extract keyring structure without sensitive information by running
+
+```sh
+lochness_check_config.py -ke /data/lochness_root/.lochness.enc
 ```
 
+
+4. Set up REDCap Data Entry Trigger if using REDCap. Please see below 
+   "REDCap Data Entry Trigger capture" section.
+
+
+5. Edit Personally identifiable information mapping table. Please seee below
+   "Personally identifiable information removal from REDCap and RPMS data"
+
+```
+/data/lochness_root/pii_convert.csv
+```
+
+
+5. Run the `sync.py` or use the example command in `2_synch_command.sh`
+
+```
+bash 2_sync_command.sh
+```
+
+
+### REDCap Data Entry Trigger capture
+
+If your sources include REDCap and you would like to configure lochness to 
+only pull new REDCap data, "Data Entry Trigger" needs to be set up in REDCap.
+
+In REDCap,
+- "Project Setup"
+- "Enable optional modules and customizations"
+- "Additional customizations"
+- check "Data Entry Trigger" and give address of the server including the port number. eg) http://pnl-t55-7.partners.org:9999
```

Comment:
Typos:

check-->Check for consistency w/ the above points?

number. eg) --> number e.g.

Reply:
I realized that I can fix the minor typos by myself :)